examples/monetization-with-stripe/README.md

```diff
@@ -20,7 +20,9 @@ This app has a paywall for a payed feature. To enable the payed feature, users n
 2. Create a [payment link](https://dashboard.stripe.com/test/payment-links)
 3. Update the link in the [PaywallNotice component](./components/PaywallNotice.tsx)
 4. Add your [API keys](https://dashboard.stripe.com/apikeys) to the [.env file](.env)
-5. For development, in your terminal:
+5. [Create a webhook endpoint](https://dashboard.stripe.com/test/webhooks/create?endpoint_location=local) in Stripe (_test in a local environment_) and download the Stripe CLI tool:
```

Comment:
I don't remember if I created a webhook. I only did the steps below with a different URL for step 2:
<img width="491" alt="image" src="https://github.com/miroapp/app-examples/assets/1961590/8ed80ebf-787d-41cb-aa7d-6f982680283e">

Reply:
I know it's been a long time since this PR has been looked at - does it need to be merged or should we close it without merging @kirillsud ?

Reply:
I just checked the manual one more time. I think it is valid

Reply:
but it has a conflict with main branch